src/components/CioQuiz/index.tsx

```diff
@@ -31,13 +31,19 @@ export default function CioQuiz(props: IQuizProps) {
   const [showSessionPrompt, setShowSessionPrompt] = useState(false);
   const { resultsPageOptions, sessionStateOptions } = props;
 
+  const skipToResults = isQuizCompleted() && !sessionStateOptions?.showSessionModalOnResults;
```

Comment:
I think all we care about here is whether we will `skipToResults` or not. We don't care about exposing `isQuizCompleted` or `hasSessionStorageState` 

What do you think if we moved this logic inside the `useQuizLocalState` hook and only returned whether we should `skipToResults` or not?

Reply:
I like that 👍